src/vsr/superblock.zig

```diff
@@ -863,6 +873,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
             vsr_state.commit_max = update.commit_max;
             vsr_state.sync_op_min = update.sync_op_min;
             vsr_state.sync_op_max = update.sync_op_max;
+            vsr_state.sync_view = update.sync_view;
```

Comment:
We could assert that `update.sync_view == 0 or update.sync_view == superblock.staging.vsr_state.sync_view`.